app/Libraries/Multiplayer/Mod.php

```diff
@@ -118,10 +119,9 @@ class Mod
             self::HALFTIME,
             self::DAYCORE,
             self::NIGHTCORE,
-        ],
-        [
```

Comment:
The removal of these brackets is intentional because as far as I can tell this was always wrong. {Half,Double} Time, {Day,Night}core and Wind {Up,Down} were always mutually exclusive and should have been in one incompatibility group all along. Adaptive Speed is incompatible with them all, so this both needed for correctness and sort of killing two birds with one stone.